continuous_integration/travis/install.sh

```diff
@@ -61,14 +62,10 @@ install_conda()
 install_pip()
 {
     # Install PyQt5
-    conda install 'pyqt>=5.6.0';
+    pip install pyqt5;
 
     # Install testing packages
-    pip install pytest pytest-cov pytest-qt mock
-
-    # Install extra packages
-    EXTRA_PACKAGES="matplotlib pandas sympy pyzmq pillow"
-    pip install $EXTRA_PACKAGES
+    pip install -e .[test]
```

Comment:
This is not needed here. Please do it after the wheel has been built, i.e. here:

https://github.com/spyder-ide/spyder/blob/master/continuous_integration/travis/build_test.sh#L10